src/YOUR_NAME_ex/pa01/snakes_and_ladders.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+
+__authors__ = 'Kaspar Akilles Lilja, Kevin Martin Lankut'
+__emails__ = 'kalilja@nmbu.no, kela@nmbu.no'
+
+"""
+Source:
+https://codereview.stackexchange.com/questions/176586/snakes-and-ladders-game
+"""
+
+import random
+import statistics
+
+
+def move_player(player, current_pos):
+    snake_squares = {24: 5, 33: 3, 42: 30, 56: 37, 64: 27, 74: 12, 87: 70}
+    ladder_squares = {1: 40, 8: 10, 36: 52, 43: 62, 49: 79, 65: 82, 68: 85}
+
+    throw = random.randint(1, 6)
+    next_pos = current_pos + throw
+    if next_pos in snake_squares:
+        next_pos = snake_squares[next_pos]
+    elif next_pos in ladder_squares:
+        next_pos = ladder_squares[next_pos]
+    return next_pos
+
+
+def single_game(num_players):
+    players = {}
+    for player in range(1, num_players + 1):
+        players[player] = 0
+
+    # Start game
+    start = 1
```

Comment:
this start value is a bit odd. I would call it something like `won = False`. That is a bit more explicit.